checkov/common/bridgecrew/integration_features/features/vulnerabilities_integration.py

```diff
@@ -0,0 +1,186 @@
+from checkov.sast.report import SastData, SastReport
+from checkov.sca_package_2.consts import get_package_by_str, ScaPackageFile, sca_package_to_sast_lang_map
+
+if TYPE_CHECKING:
+    from checkov.common.bridgecrew.platform_integration import BcPlatformIntegration
+    from checkov.common.output.report import Report
+    from checkov.common.typing import _BaseRunner
+
+NORMALIZE_PREFIX = 'BC_NORMALIZE_'
+
+
+class VulnerabilitiesIntegration(BaseIntegrationFeature):
+    def __init__(self, bc_integration: BcPlatformIntegration) -> None:
+        super().__init__(bc_integration=bc_integration, order=2)  # must be after the custom policies integration
+        # the provider may be lower or upper depending on where the policy was created
+
+    def is_valid(self) -> bool:
+        return (
+            self.bc_integration.is_integration_configured()
+            and not self.bc_integration.skip_download
+            and not self.integration_feature_failures
+        )
+
+    def pre_scan(self) -> None:
+        # not used
+        pass
+
+    def pre_runner(self, runner: _BaseRunner) -> None:
+        # not used
+        pass
+
+    def post_runner(self, scan_report: Report) -> None:
+        # not used
+        pass
+
+    def post_scan(self, merged_reports: list[Report]) -> None:
+        self.merge_sca_and_sast_reports(merged_reports)
+
+    def merge_sca_and_sast_reports(self, merged_reports: list[Report]) -> None:
+        if not bool(convert_str_to_bool(os.getenv('CKV_ENABLE_UPLOAD_SAST_IMPORTS', False))) or \
+                not bool(convert_str_to_bool(os.getenv('CKV_ENABLE_SCA_INTEGRATE_SAST', False))):
+            return
+
+        # Extract SAST imports report
+        sast_reports = [scan_report for scan_report in merged_reports if type(scan_report) == SastReport]
+        if not len(sast_reports):
+            return
+
+        sast_imports_report = SastData.get_sast_import_report(sast_reports)
+
+        # Extract SCA packages report
+        sca_packages_report = [scan_report for scan_report in merged_reports if
+                               scan_report.check_type == CheckType.SCA_PACKAGE]
+        if not len(sca_packages_report):
+            return
+
+        # Extract vulnerabilities failed checks
+        cves_checks = [cve_check for cve_check in sca_packages_report[0].failed_checks if
+                       cve_check.check_name == SCA_PACKAGE_SCAN_CHECK_NAME]
+
+        # Create a dictionary to store the grouped records
+        grouped_records = self.group_cves_checks_by_files(cves_checks)
+
+        # Iterate over each file, get sast imports and enrich it's Cves
+        for sca_file_path, current_cves in grouped_records.items():
+
+            # Determine the langauge of file
+            lang = self.get_sast_lang_by_file_path(sca_file_path)
+
+            # Extract Sast data from Sast report filtered by the language
+            entries = sast_imports_report.get('imports', {}).get(lang, {}).items()
```

Comment:
let's create assets fields name (e.g. `imports`), @matansha can help with that

Reply:
I agree it's better, but we can do this refactor after my PR.